hikey960.mk

```diff
@@ -0,0 +1,444 @@
+################################################################################
+# Following variables defines how the NS_USER (Non Secure User - Client
+# Application), NS_KERNEL (Non Secure Kernel), S_KERNEL (Secure Kernel) and
+# S_USER (Secure User - TA) are compiled
+################################################################################
+COMPILE_NS_USER   ?= 64
+override COMPILE_NS_KERNEL := 64
+COMPILE_S_USER    ?= 32
+COMPILE_S_KERNEL  ?= 64
+
+# Normal and secure world console UART: 6 (v2 or newer board) or 5 (v1 board)
```

Comment:
Doh! Did they really manage to produce incompatible versions of the board already? Anyway...